getting_started/data_backup.md

```diff
@@ -0,0 +1,220 @@
+Users may need to use a variety of methods to meet their backup/file transfer needs.
+
+> We **STRONGLY RECOMMEND** users configure [automated backups](#automated-backups)
+
+> Windows Users: `scp` and `rsync` are not installed by default on Windows.
+> You will need to install them manually or use [WSL](https://docs.microsoft.com/en-us/windows/wsl/about)
+
+[RAID]: https://en.wikipedia.org/wiki/RAID
+
+## Small Transfers: `scp`
+
+For small transfers (< 1GB) on to or off of arjuna, the following `scp` commands an be used from *your local machine*.
+
+> Remember to replace `user` with your username
+
+| Action                    | Command                                             |
+| ------------------------- | --------------------------------------------------- |
+| Copy a File from Arjuna   | `scp user@arjuna.psc.edu:path/to/file path/to/dst`  |
+| Copy a File to Arjuna     | `scp path/to/file user@arjuna.psc.edu:path/to/dst`  |
+| Copy a Folder from Arjuna | `scp -r user@arjuna.psc.edu:path/to/folder path/to` |
+| Copy a Folder to Arjuna   | `scp -r path/to/folder user@arjuna.psc.edu:path/to` |
+
+Notice when copying folders, the destination folder is `/path/to` not `/path/to/folder`.
+`scp` will copy the folder to `/path/to` thus creating `/path/to/folder`; using a destination of `/path/to/folder` would result in `/path/to/folder/folder`.
+
+These commands can be shortened to `scp arjuna:/path/to/file /path/to/dst` by
+setting up an [SSH config file](../getting_started/connecting.md#using-a-ssh-config-file).
+To read the manual page for `scp`, type `man scp` into a terminal.
+
+## Managing Code
+
+Source code track by [git] can be transferred via [GitHub] directly via ssh:
+
+```bash
+git clone git@github.com:ArjunaCluster/ArjunaUsers.git
+```
+
+See GitHub's instruction for connecting to [GitHub via ssh](https://docs.github.com/en/authentication/connecting-to-github-with-ssh) for more information.
+
+Additional resources for git and GitHub can be found on our [Other Resources](./linux.md#git) page.
+
+[git]: https://git-scm.com
+[GitHub]: https://github.com
+
+## Larger Transfers: `rsync`
+
+For larger file transfers (> 1GB or > 20 files), the following `rsync` commands an be used from *your local machine*.
+
+> Remember to replace `user` with your username
+
+
+| Action                    | Command                                             |
+| ------------------------- | --------------------------------------------------- |
+| Copy a File from Arjuna   | `rsync user@arjuna.psc.edu:path/to/file path/to/dst`  |
+| Copy a File to Arjuna     | `rsync path/to/file user@arjuna.psc.edu:path/to/dst`  |
+| Copy a Folder from Arjuna | `rsync -r user@arjuna.psc.edu:path/to/folder path/to` |
+| Copy a Folder to Arjuna   | `rsync -r path/to/folder user@arjuna.psc.edu:path/to` |
+
+Unlike `scp`, `rsync` checks files sizes and modification times to only transfer files that have changed. This is helpfully for several reasons:
```

Comment:
helpful

Reply:
This should be this is helpful because it ... or the form of the bullet points should change to make it make grammatical sense